torch/csrc/jit/script/compiler.cpp

```diff
@@ -223,6 +223,26 @@ struct Environment {
       }
     }
   }
+
+  // for each captured input, create/refer new one in enclosing
+  // scope, and replace references in the block, delete branch block
+  // inputs, and the corresponding field in value_map
+  void deleteIfCapturedInputs(const SourceRange& loc) {
```

Comment:
this name is actually ambiguous, it's not clear if `if` refers to an `if` statement or it's saying "delete if we have captured inputs". Can we name this something like `deleteIfStmtCapturedInputs`?